src/styles/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  color: var(--dark);
+  margin-bottom: 4px;
+  font-size: 12px;
+
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  color: var(--grey);
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-count {
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+  line-height: 14px;
+}
+
+.price__price-tag {
+  color: var(--grey);
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.price__price-amount {
+  color: var(--dark);
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__button {
+  box-sizing: border-box;
+  display: flex;
+
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+
+  background: var(--blue);
+  border: 1px solid transparent;
+  border-radius: 5px;
+  color: #fff;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-decoration: none;
+}
+
+.card__button:hover {
+  background: white;
+  color: var(--blue);
+  border-color: var(--blue);
+
+}
+
+p {
+  margin: 0;
+}
```

Comment:
Don't do it here. If you want to reset some styles - do it in the main css file or in separate `reset.css` file.